RijsMuseum/app/src/main/java/com/example/rijsmuseum/viewmodel/DetailsViewModel.kt

```diff
@@ -1,10 +1,23 @@
 package com.example.rijsmuseum.viewmodel
 
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.example.network.getters.CollectionsDetailsGetter
+import com.example.network.models.CollectionsDetails
 
 class DetailsViewModel: ViewModel() {
-    fun getCollectionsDetailsRequest(data: CollectionsDetailsGetter.DataReadyCallback) {
-        CollectionsDetailsGetter().getCollectionsDetailsRequest(data)
+    var _dataTitle = MutableLiveData<String>()
```

Comment:
It is not a good practice to have MutableLiveDatas as public properties. You should change their values only inside of the viewModel.